packages/amplify-e2e-tests/src/__tests__/amplify-app.test.ts

```diff
@@ -17,6 +17,7 @@ import {
   validateAmplifyPush,
   validateFeatureFlags,
 } from '../amplify-app-helpers/amplify-app-validation';
+import { AmplifyFrontend } from 'amplify-cli-core';
```

Comment:
Does this import need to be updated?

```suggestion
import { AmplifyFrontend } from '@aws-amplify/amplify-cli-core';
```